### PR TITLE
1.4 initial docs

### PR DESCRIPTION
**Pull request by ES-Alexander**
# 🔎 [PREVIEW](http://br-www-docs.s3-website-us-east-1.amazonaws.com/preview/blueos/44/usage/overview/) 🔍

**Documents:**
- bluerobotics/BlueOS#2873
- bluerobotics/BlueOS#2877
- bluerobotics/BlueOS#2930
- bluerobotics/BlueOS#3021
- bluerobotics/BlueOS#2536
- bluerobotics/BlueOS#2912
- bluerobotics/BlueOS#2988
- bluerobotics/BlueOS#2909
- bluerobotics/BlueOS#2890
- bluerobotics/BlueOS#2888
- bluerobotics/BlueOS#2857
- bluerobotics/BlueOS#2965
- bluerobotics/BlueOS#2983
- bluerobotics/BlueOS#2995
- bluerobotics/BlueOS#3023
- bluerobotics/BlueOS#2935

**Minimally documents:**
- bluerobotics/BlueOS#2280
    - Would be nice to provide one or more examples
- bluerobotics/BlueOS#2815
    - Would be nice to provide one or more examples
    - Ideally we'd provide a tool/process for checking whether relative path access actually works properly
- bluerobotics/BlueOS#3028
    - Would be nice to provide one or more examples of how the field can be used
- bluerobotics/BlueOS#3004
    - Tools in general could do with their own table and explanations, instead of just basic links from services that use them

### CODE REVIEW

File: content/usage/advanced/index.md [hunk 1]
```diff
@@ -582,13 +614,16 @@ including failsafes, and reverting parameters to their defaults.
...
+
 ### Video Streams
 {{ service(service="MAVLink Camera Manager", link="https://github.com/bluerobotics/mavlink-camera-manager/", port=6020) }}
 
 - BlueOS automatically detects H264-encoded video streams
 {% pirate() %}
-   - MJPG and YUYV encoded streams are also detected in pirate mode,
-   but currently only work when configured as RTSP streams
+- MJPG and YUYV encoded streams are also detected in pirate mode, but currently only work when configured as RTSP streams
+- H265 encoded streams can be streamed using UDP265 `(New in 1.4)`
```
Comment (joaoantoniocardoso) [inline content/usage/advanced/index.md]: or RTSP

File: content/usage/advanced/index.md [hunk 2]
```diff
@@ -205,6 +222,8 @@ Submit feedback about BlueOS via:
 - [Issues on the GitHub repository](https://github.com/bluerobotics/BlueOS/issues)
    - allows easily tracking changes, 
 and notification when complete/fixed
+- Report form submissions
+   - Can be submitted at any time, and get sent to the developers when the vehicle next connects to the internet
```
Comment (patrickelectric) [inline content/usage/advanced/index.md]: surface computer is connected to the internet

Comment (ES-Alexander) [reply content/usage/advanced/index.md]: Ahh, makes sense, thanks :-)

File: content/usage/advanced/index.md (hunk 1 above)
Comment (ES-Alexander) [reply content/usage/advanced/index.md]: Good to know, thanks! :-)

File: content/usage/advanced/index.md [hunk 3]
```diff
@@ -268,6 +289,8 @@ The Autopilot Parameters page allows checking and changing the autopilot's confi
 
 - Includes fuzzy searching of names and descriptions, to help find relevant parameters
 - Allows loading parameters from a file, and saving the current parameters to a file
+- Parameter descriptions can be overridden by including a `metatdata_override.json` file
```
Comment (Williangalvani) [inline content/usage/advanced/index.md:291]: it would be nice to link to how these are created (the python script that generates the xml/json/rst files)

**Pull request merged**